content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,265 @@
+which are used to show how to instrument a distributed system with
+OpenTelemetry.
+
+The main focus of the web app is to demonstrate how to instrument an
+application, no matter what programming language, platform or OS it uses. The
+web app also shows different instrumentation techniques: automatic and manual,
+metrics, and baggage. All while following the standards and conventions
+proscribed in the official OTel documentation. More about the specific
+requirements can be
+[found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
+
+My company was focused on becoming part of and embracing the OpenTelemetry
+community. One of our goals this summer was to get more involved with a core
+OpenTelemetry project where we could provide a meaningful contribution. The OTel
+demo was the best match for our goal, as contributing would not only help the
+community, but also provide a great example to test and showcase our product
+with.
+
+The first thing we did was to get in contact with
+[Carter Socha](https://github.com/cartersocha), the organizer of the OTel Demo
+SIG. Carter was really welcoming and helped us identify where our contributions
+could be the most impactful. We started looking at
+[the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
+referencing a complete front-end overhaul that would involve moving the
+application away from Go server-side render (SSR) to an architecture that
+included a browser-side client (client-side render or CSR), as well as improving the
+overall style, theme, and user experience.
+
+A fun aspect of the work was the request to move the store from a "normal" store
+to an astronomy store to match the OpenTelemetry project’s overall branding.
+
+Once we got the green light from the rest of the OTel demo SIG, then we started
+working on the different changes that were part of the front-end architecture
+overhaul.
+
+## OpenTelemetry Demo Application Description and Tech Stack
+
+The demo app is an astronomy store, with basic eCommerce features such as a
+shopping cart, currency selector, payment, and checkout. It also includes the
+ability to display promotions (ads) and related products depending on the user’s
+context.
+
+The demo’s stack includes multiple microservices in different languages,
+covering each of the following OTel-supported languages:
+
+- .NET
+- C++
+- Elixir
+- Go
+- Node.js
+- PHP
+- Python
+- Ruby
+- Rust
+
+Every microservice has a specific goal and can communicate with others by using
+a global gRPC definition. Persistent information is saved into a PostgresSQL
+database and there are outbound services that connect with third-party services
+to trigger events (such as confirmation emails). All of the microservices,
+including the front-end, are connected to the same OpenTelemetry collector
+instance, which uses Jaeger as one of the data stores for the tracing data.
+
+```mermaid
```

Comment:
while github has a mermaid support, we do not have it in hugo, so you still need to put the picture here.

Reply:
Ups, sorry about that, I just fixed it with my latest commit. Thanks for the heads up! 😄